pkg/cmd/pulumi/package_extract_schema.go

```diff
@@ -25,16 +25,18 @@ import (
 
 func newExtractSchemaCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "get-schema <schema_source>",
-		Args:  cobra.ExactArgs(1),
+		Use:   "get-schema <schema_source> [provider parameters]",
```

Comment:
I would go with a repeatable flag here, not arbitrary args:

```
pulumi package get-schema terraform-bridge --param name=random -p version='>=3'
```

Instead of 
```
pulumi package get-schema terraform-bridge name=random version='>=3'
```

Reply:
Seems a lot more verbose, not clear what the win is there?

Reply:
yeh gonna go with this feels more natural to me for now